### PR TITLE
Asset comment settings

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?

This PR adds

- Coral UI 
  - Checkbox
- Configure Stream Tab Functionality
- UX/UI

Note: 'Pre-Moderate Comments Containing Links' is ready on the frontend, once we have that in the backend, we'll tie that up.

## How to test this?

- Go to the comment stream
- Go to the third tab, Configure Stream
- Configure the stream 
- Click some checkbox, they will turn green and as soon as you hit apply the will turn back grey. 💃 (approved by Emma )

### CODE REVIEW

**Comment by okbel** [comment]
Would you mind taking a look at this one, @davidgljay and @wyattjoh? //cc @kgardnr

File: routes/api/asset/index.js
Comment (wyattjoh) [inline routes/api/asset/index.js]: Updates can not return a payload, this should be reverted to return 204 without a body.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

**Comment by wyattjoh** [comment]
This should not be using the `Asset.status` field, and instead should defer to what we have in the close-comments (https://github.com/coralproject/talk/tree/close-comments) branch.

**Comment by okbel** [comment]
@wyattjoh I'll update the PR as soon as `close-comments` branch gets merged

File: client/coral-configure/components/ConfigureCommentStream.js
Comment (davidgljay) [inline client/coral-configure/components/ConfigureCommentStream.js]: This and other text should be handled via i18n.

File: client/coral-configure/components/ConfigureCommentStream.css
Comment (davidgljay) [inline client/coral-configure/components/ConfigureCommentStream.css:1]: Earlier we made a decision to handle all comment stream css in default.css for ease of reconfigurability. I'm open to revisiting that decision (it probably generally makes sense to reexamine our strategy for customizing style), but we should be intentional about it before changing norms.

File: client/coral-framework/reducers/config.js
Comment (davidgljay) [inline client/coral-framework/reducers/config.js:2]: This makes sense as a standard, but it's not reflected for items and notifications. Should we make that update here or put it in a separate PR? If we're going to refactor we should make it consistent throughout.

**Review by davidgljay — CHANGES_REQUESTED** [review]
One change requested around i18n and a few standards questions.

File: client/coral-configure/components/ConfigureCommentStream.css
Comment (okbel) [reply client/coral-configure/components/ConfigureCommentStream.css:1]: I agree. The idea of having the `default.css` was for customizing and creating themes. This idea still remains. But components need to have core styling such as positions, and vertical-aligns. The app must still work regardless of the `default.css`. That being said, shouldn't the work of `default.css` only be for theming and overriding purposes?

**Comment by okbel** [comment]
@wyattjoh @davidgljay Everything updated. Thanks!

**Review by gabelula — APPROVED** [review]
Ready to merge.

**Review by gabelula — APPROVED** [review]

**Review by gabelula — APPROVED** [review]

**Pull request merged**
